astrapy/idiomatic/collection.py

```diff
@@ -216,6 +216,9 @@ def delete_many(
                 f"(gotten '${json.dumps(dm_response)}')"
             )
 
+    def drop(self, name_or_collection: Union[str, AsyncCollection]) -> None:
```

Comment:
As a method on collection this has no arguments, it is implied that [self is going to be dropped](https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.drop).